app/src/main/res/layout-land/profile_edit_fragment.xml

```diff
@@ -177,7 +177,7 @@
         style="?android:attr/borderlessButtonStyle"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginBottom="@dimen/profile_edit_delete_margin_bottom"
+        android:layout_marginBottom="12dp"
```

Comment:
Is there a need to change the margin bottom here?
If we are changing it then how about updating the value of the `dimen/profile_edit_delete_margin_bottom`.

Reply:
Sure will change the value of it